invenio_oauthclient/templates/semantic-ui/invenio_oauthclient/_macros.html

```diff
@@ -0,0 +1,43 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2018 CERN.
```

Comment:
Very minor, everywhere
```suggestion
  Copyright (C) 2015-2020 CERN.
```